### ISSUE TITLE
Add support for all standard datatypes

### ISSUE THREAD

**Issue by buijs-dev** [buijs-dev/klutter#11]
TLDR: Maps/arrays are not supported right now which means any KlutterJSON implementation using a map will not be converted to a Dart DTO. 

Solution:
Expand the DartKotlinMap with the following types and add support for code generation based on these types:
INTEGER("Int", "int"),
DOUBLE("Double", "double"),
BOOLEAN("Boolean", "bool"),
STRING("String", "String"),
NOTHING("Unit", "void"),
LONG("Long", "int"),
BYTE_ARRAY("Uint8List","ByteArray"),
INT_ARRAY("Int32List","IntArray"),
LONG_ARRAY("Int64List","LongArray"),
FLOAT_ARRAY("Float32List","FloatArray"),
DOUBLE_ARRAY("Float64List", "DoubleArray"),
LIST("List", "List"),
MAP("Map", "HasMap");

**Comment by buijs-dev** [comment]
All standard types are supported in 2023.1.1.beta (dart 0.3.0) and newer.
